ui/src/app/components/side-nav.tsx

```diff
@@ -407,7 +407,7 @@ export const SideNav = (props: SideNavProps) => {
           <SideNavItem
             content='Workspaces'
             onToggleSideNav={() => onToggleSideNav()}
-            href='admin/workspaces'
+            href='/admin/workspaces'
```

Comment:
This worked already because the underlying component works differently, but updated for consistency